HappyAnding/HappyAnding/Views/Components/AdminCurationCell.swift

```diff
@@ -7,14 +7,74 @@
 
 import SwiftUI
 
+/*
```

Comment:
```suggestion 
/**
```

문서화할 때는 여러줄 주석을 /** */으로 나타냅니다!

Reply:
오 처음 알았습니다! 이제부터 적용해보겠습니다 :)